--- a/Dockerfile
+++ b/Dockerfile
@@ -65,3 +65,3 @@ COPY src/w64devkit.c src/w64devkit.ico \
 
-ARG ARCH=x86_64-w64-mingw32
+ARG ARCH=i686-w64-mingw32
 
@@ -93,2 +93,3 @@ RUN /mingw-w64-v$MINGW_VERSION/mingw-w64-headers/configure \
         --with-default-msvcrt=msvcrt-os \
+        --with-default-win32-winnt=0x0501 \
  && make -j$(nproc) \
@@ -99,2 +100,5 @@ RUN ln -s $ARCH mingw
 
+# Disable UTF-8 manifest for Windows XP (#58)
+RUN echo >/gcc-$GCC_VERSION/gcc/config/i386/winnt-utf8.manifest
